modules/pam_motd/pam_motd.8.xml

```diff
@@ -64,8 +64,9 @@
       override files with the same name in <filename>/usr/lib/motd.d/</filename>.
     </para>
     <para>
-      Files the in the directories listed above are displayed in
-      lexicographic order by name.
+      Files in the directories listed above are displayed in lexicographic
+      order by name. Moreover, the files are filtered by checking the ownership
```

Comment:
I'd use something like: "files are filtered by reading them with the credentials of the target user authenticating on the system."